migrations/schemas/20220611111531-message_repost_histories.sql

```diff
@@ -0,0 +1,16 @@
+
+-- +migrate Up
+create table message_repost_histories
```

Comment:
why do we need to store repost history

Reply:
i want to track that a message was already reposted or not, so when users keep reacting/removing on that message, it'll not be reposted again